package.json

```diff
@@ -8,6 +8,7 @@
     "dist/**/*"
   ],
   "scripts": {
+    "benchmarks": "npm run build && ts-node benchmarks/index.ts",
```

Comment:
Ah, didn't see this when doing the comment on the benchmark files above.

However this fails for me with a completely fresh node module reinstall with several implicit `any` type errors. Any comment on that?

Reply:
Thanks, I just did a fresh install and ran into the errors, will look at fixing.

Reply:
Can confirm that this works now.